internal/cmd/gocdk/launch.go

```diff
@@ -172,30 +169,16 @@ func (local *localLauncher) Launch(ctx context.Context, input *LaunchInput) (*ur
 	} else if hostPort < 0 || hostPort > 65535 {
 		return nil, xerrors.Errorf("local launch: host_port is out of range [0, 65535]")
 	}
-	dockerArgs := []string{
-		"run",
-		"--rm",
-		"--detach",
-		"--publish", fmt.Sprintf("%d:8080", hostPort),
-	}
-	for _, v := range input.Env {
-		dockerArgs = append(dockerArgs, "--env", v)
-	}
-	dockerArgs = append(dockerArgs, "--env", "PORT=8080")
-	dockerArgs = append(dockerArgs, input.DockerImage)
-
-	c := exec.CommandContext(ctx, "docker", dockerArgs...)
-	c.Env = local.dockerEnv
-	c.Dir = local.dockerDir
```

Comment:
IIUC, we lost this (setting `exec.Cmd.Dir` to `pctx.workdir`, previously via `dockerDir`. Was it not needed?

Reply:
It was not needed. The working directory that we run the `docker` CLI in is only relevant for `docker build`.